docs/api/covidcast.md

```diff
@@ -113,6 +113,16 @@ and lists.
 The current set of signals available for each data source is returned by the
 [`covidcast_meta`](covidcast_meta.md) endpoint.
 
+#### Alternate Required Params
+
+The following params help specify multiple source-signal, timetype-timevalue or geotype-geovalue pairs. Use them in replacement of the usual required params.
+
+| Parameter | Replacing | Format | Description | Example |
```

Comment:
```suggestion
#### Alternate Required Parameters

The following parameters help specify multiple source-signal, timetype-timevalue or geotype-geovalue pairs. Use them instead of the usual required parameters.

| Parameter | Replaces | Format | Description | Example |
```